### PR TITLE
Bug 1303299, CLI tools download and About page for web console

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1303299

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

**Comment by tpoitras** [comment]
I think there's gonna be a clash with this PR: https://github.com/openshift/openshift-docs/pull/1467

Whichever one merges first will force the other to rebase.

**Comment by ahardin-rh** [comment]
Fixed the merge conflicts and reworked my edits.

**Comment by adellape** [comment]
@ahardin-rh We're forgetting to take openshift-dedicated and openshift-online into account for get_started_cli.

**Comment by adellape** [comment]
Actually, that needs to get fixed before the new About page stuff here, so I'll fix that in a separate PR to get published this week.

**Comment by ahardin-rh** [comment]
In today's team meeting, it was suggested that I incorporate a screenshot. I will get that added soon.

**Comment by tpoitras** [comment]
:camera: :sparkler: :+1:

**Comment by adellape** [comment]
@ahardin-rh Just a reminder, this is unblocked now that the fix in https://github.com/openshift/openshift-docs/pull/1661 was merged/published.

**Comment by ahardin-rh** [comment]
:construction_worker: Thanks! I'm on it! :hammer:

**Comment by ahardin-rh** [comment]
@adellape Completed my latest edits. Please check behind me to see if I missed anything.  :tada:

File: cli_reference/get_started_cli.adoc [hunk 1]
```diff
@@ -36,6 +36,13 @@ for instructions per your workstation's operating system.
 
 [[installing-the-cli]]
 == Installing the CLI
+You can access the *About* page on the web console
```
Comment (adellape) [inline cli_reference/get_started_cli.adoc]: s/access/download the CLI from/

As it's written, the sentence sounds like it's just saying you can access the page but not what to do there, or that the page itself may or may not be enabled by the admin.

Suggestion: Maybe also start this with something like, "The easiest way to download the CLI is by accessing..."

File: cli_reference/get_started_cli.adoc [hunk 2]
```diff
@@ -36,6 +36,13 @@ for instructions per your workstation's operating system.
...
+ifdef::openshift-enterprise,openshift-origin[]
+if your cluster administrator has enabled it
```
Comment (adellape) [inline cli_reference/get_started_cli.adoc]: s/it/the download links/

File: cli_reference/get_started_cli.adoc [hunk 3]
```diff
@@ -195,6 +222,13 @@ $ tar -xf <file>
 
 // end::installcli[]
 
+ifdef::openshift-enterprise,openshift-origin[]
+Alternatively, if the cluster administrator has enabled it, you can download and
+unpack the CLI from the
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console].
+endif::[]
```
Comment (adellape) [inline cli_reference/get_started_cli.adoc]: I think this should be split up and put into the respective ifdefs like you have it in the Windows and Mac sections.

Comment (adellape) [inline cli_reference/get_started_cli.adoc]: Or, put this right before "Then, unpack the archive" a few lines up.

**Comment by ahardin-rh** [comment]
@adellape Thanks! Made these updates and added in the new version of the screenshot with pixelation to both sections.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../architecture/infrastructure_components/web_console.html[Web Console]

|Added a link:../architecture/infrastructure_components/web_console.html#web-console-cli-downloads[CLI Downloads] section, including a screenshot of the new *About* page. 
%
|link:../cli_reference/get_started_cli.html[Get Started with the CLI]

|In the link:../cli_reference/get_started_cli.html#installing-the-cli[Installing the CLI section], added information about downloading the CLI from the *About* page, including a screenshot of the *About* page.
%
|link:../install_config/web_console_customization.html[Customizing the Web Console]

|In the link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[Adding or Changing Links to Download the CLI] section, added information about downloading the CLI from the *About* page.
%

**Pull request merged**